src/uct/ib/rdmacm/rdmacm_iface.c

```diff
@@ -147,6 +148,8 @@ static void uct_rdmacm_iface_process_conn_req(uct_rdmacm_iface_t *iface,
     }
 
     cb_hdr.length = server_data_len;
+    /* The private_data starts with the header of the user's private data and then
+     * the private data itself */
     memcpy((void*)conn_param.private_data, &cb_hdr.length, sizeof(uct_rdmacm_priv_data_hdr_t));
```

Comment:
Ok, looks like i also need to be more clear :) pls add compile time check for `sizeof(cb_hdr.length) == sizeof(uct_rdmacm_priv_data_hdr_t)`

Reply:
@evgeny-leksikov isn't this a trivial check? becuase length is the only field in uct_rdmacm_priv_data_hdr_t. Can you pls explain why you think it's needed?

Reply:
It's easy to break adding new field to uct_rdmacm_priv_data_hdr_t structure of changing size of `cb_hdr.length` or `uct_rdmacm_priv_data_hdr_t.length`, for example for optimization needs.

Reply:
you are copying from one buffer to another by size of third.